DQMOffline/Muon/interface/CSCOfflineMonitor.h

```diff
@@ -231,12 +231,12 @@ class CSCOfflineMonitor : public DQMEDAnalyzer {
   MonitorElement *hSensitiveAreaEvt;
 
   // BX monitoring
-  MonitorElement *hALCTgetBX;
-  MonitorElement *hALCTgetBXSerial;
-  //  MonitorElement *hALCTgetBXChamberMeans;
-  //  MonitorElement *hALCTgetBX2DMeans;
-  MonitorElement *hALCTgetBX2Denominator;
-  MonitorElement *hALCTgetBX2DNumerator;
+  MonitorElement *hALCTbx;
+  MonitorElement *hALCTbxSerial;
+  //  MonitorElement *hALCTbxChamberMeans;
```

Comment:
These name changes will probably need a PR in 
https://github.com/dmwm/deployment/blob/master/dqmgui/style/CSCRenderPlugin.cc#L2684-L2686
to mantain the same style in DQM GUI

Reply:
@jfernan2 I'll undo the changes in the histogram naming. Thanks for pointing this out.